code/game/objects/structures/crates_lockers/closets/secure/security.dm

```diff
@@ -95,6 +95,8 @@
 	new /obj/item/storage/belt/security/full(src)
 	new /obj/item/flashlight/seclite(src)
 	new /obj/item/door_remote/head_of_security(src)
+	new /obj/item/gun/ballistic/automatic/pistol/sp8/nomag(src)
```

Comment:
Why is this spawning magless when you could just spawn one with a mag and take out the additional magazine in the locker.

Reply:
I though firearms should be stored unloaded, but that's easy to change